.gitignore

```diff
@@ -3,3 +3,4 @@
 dist/
 .DS_Store
 **/.DS_Store
+.idea
```

Comment:
As this doesn't depend specifically on this repository and each editor would require different folders or files to ignore, what about adding that to the personal global ignore file (i.e. `~/.gitignore`)?

Reply:
Sounds good to me - I'm closing this MR.